scripts/lls-check

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+# Performs a lua-language-server check on all files.
+# lls-check/check.json will be produced on any issues, returning 1.
+# Outputs check.json to stdout, all messages to stderr, to allow jq etc.
+
+VER_LLS="3.7.3"
+
+DIR_SRC="${PWD}"
+DIR_OUT="${DIR_SRC}/lls-out"
+DIR_LLS="/tmp/lls"
+
+# clear output
+rm -rf "${DIR_OUT}"
+mkdir "${DIR_OUT}"
+
+if [ ! -d "${DIR_LLS}" ]; then
+	# download lua-language-server binaries
+	mkdir -p "${DIR_LLS}"
+	curl -L "https://github.com/LuaLS/lua-language-server/releases/download/${VER_LLS}/lua-language-server-${VER_LLS}-linux-x64.tar.gz" | tar zx --directory "${DIR_LLS}"
```

Comment:
Ubuntu doesn't have a package and there don't seem to be any other means to source and run this.